implementation/endpoints/src/netlink_connector.cpp

```diff
@@ -17,6 +17,15 @@
 
 namespace vsomeip_v3 {
 
+namespace {
+    const std::uint32_t max_retries = 3;
+    const std::uint32_t retry_bit_shift = 8;
+    const std::uint32_t request_sequence_bitmask = 0xFF;
+    const std::uint32_t ifa_request_sequence = 1;
+    const std::uint32_t ifi_request_sequence = 2;
+    const std::uint32_t rt_request_sequence = 3;
+}
```

Comment:
Move these variables to netlink_connector.hpp as a static const member.

Reply:
Done :heavy_check_mark: